report/6.Conclusion/Futurework.tex

```diff
@@ -0,0 +1,61 @@
+In the actual implementation this was not done, as a different solution proved better.
+However to hit a projectile, this would be a requirement.
+The initial idea was to train the model by simply moving the object in front of a camera and tracking how the object moved.
+This could lead to an additional problem, based on different movement of different object types.
+Here classification could be a solution, as different objects have different movement patterns, unless the system is still based on simple inanimate objects.
+
+These aspects could be an interesting way to explore online learning and clustering so movement patterns, for instance a ball bouncing against the ground, could be learned in real time. 
+In this context, calibration could potentially also be done online, however depending on the physical design, calibration might not be relevant, from a solution aspect.
+
+
+
+\subsection{Firing a projectile}
+During our MosCow analysis, the ability to hit a moving target with a projectile was assigned as a \textbf{Could have}.
+The ability to do this requires further development of the physical aspects of the solution.
+Creating a device that can fire a projectile, and ideally not require a manual reload between each shot, requires a larger construction, which would put more weight on each motor, which would reduce precision.
+Furthermore, the solution would require an prediction algorithm.
+Prediction where to shoot would rely on the following data, as explained in section~:
+\begin{enumerate}
+	\item speed of the projectile
+	\item distance to the target
+	\item speed of the projectile
+\end{enumerate}
+This would, therefore, require more sensors, which would add levels of abstraction to the physical domain.
+This would also require more data analysis, making the calculations more abstract, than the current model of a two-axis system.
+
+
+\subsection{Machine Learning instead of color detection}
+
+Currently the device uses OpenCV color detection, as described in Section~\ref{solution:objfillalgo}.
+There were a multitude of reasons for this, the primary one being that the hardware available, a Raspberry Pi, was simply not fast enough, as described in Section~\ref{des:sec:performance}.
+However, the current solution has a couple of problems.
+
+The original reasoning behind the project was to prevent bird strikes at airports, wherein the ability target the bird but not a plane, would be an essential requirement, when implementing the device for real world use.
+Another issue is that the current solution would target a student wearing a red shirt, which could potentially be hazardous.
+
+This could be solved by using a machine learning solution, potentially the on used in Section~\ref{des:sec:performance}, however this would require a more powerful hardware.
+Potentially a custom neural network would have to be created, focusing more on the specific type of target, unlike YOLO which has multiple classification types.
```

Comment:
```suggestion
Another solution could be the implementation of a custom neural network that is focused on the specific type of target, unlike YOLO which has multiple classification types.
```

Reply:
Tror jeg skriver: "A different solution could be ...."